js/index.js

```diff
@@ -956,18 +1137,11 @@ function handleKeyPress(event) {
         case "KeyH":
             historyButton.click();
             if (historyCheckbox.checked) {
-                const firstEntry = historyList.firstElementChild;
-                if (firstEntry) {
-                    const explanationButton = firstEntry.querySelector(`button.explanation-button`);
-                    explanationButton.dispatchEvent(new Event("mouseenter"));
-                    dehoverQueue.push(() => {
-                        explanationButton.dispatchEvent(new Event("mouseleave"));
-                    });
-                }
-            } else {
-                dehoverQueue.forEach(callback => {
-                    callback();
-                });
```

Comment:
You removed a property of the 'H' key, which is that it shows you your last problem's visual explanation.